modules/shared/types/postFeed/IPostFeed.d.ts

```diff
@@ -0,0 +1,55 @@
+import type { EPostType } from './EPostType';
```

Comment:
I remember we talked about writing FE types or use the client Types, can someone explain to me why can't we use the client types? it is not a blocker for this PR. I am just asking for better understanding